### ISSUE TITLE
test Tensorflow1

### ISSUE THREAD

**Issue by sheena417** [sheena417/python#2]
# Tensorflow関連ライブラリのインポート
from tensorflow.examples.tutorials.mnist import input_data
import tensorflow as tf
import sys
import os
import subprocess

# main関数
if __name__ == "__main__":
    # 教師データのダウンロード
    mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
    # 変数宣言 変数の型、次元数を定義する　None=任意の長さ*784次元のデータ
    with tf.name_scope("read_data"):
        x = tf.placeholder(tf.float32, [None, 784])
    # 重みの初期化 784*10次元のデータ
    with tf.name_scope("Weight"):
        W = tf.Variable(tf.zeros([784,10]))
    # 閾値の初期化 10次元のデータ
    with tf.name_scope("bias"):
        b = tf.Variable(tf.zeros([10]))

    # ソフトマックス回帰によるモデル作成　matmul=入力データと重みの内積計算
    y = tf.nn.softmax(tf.matmul(x, W) + b)

    # 誤差関数の値入力用変数の宣言
    y_ = tf.placeholder(tf.float32, [None, 10])

    # セッションの作成(セッション=機械学習の流れ)
    sess = tf.Session()

    # 誤差関数の定義
    with tf.name_scope("cross_entropy"):
        cross_entropy = tf.reduce_mean(-tf.reduce_sum(y_ * tf.log(y), reduction_indices=[1]))
    # 勾配降下法を利用した誤差の算出
    with tf.name_scope("training"):
        train_step = tf.train.GradientDescentOptimizer(0.5).minimize(cross_entropy)

    # セッションの初期化と入力データの準備
    sess.run(tf.global_variables_initializer())

    # 可視化処理
    entropy_summary = tf.summary.scalar("cross_entropy", cross_entropy)
    merged = tf.summary.merge_all()
    with tf.summary.FileWriter("./logs", sess.graph) as writer:

        # 学習(1000回)
        for i in range(1000):
            # 教師データからランダムに100個のデータを取得する
            batch_xs, batch_ys = mnist.train.next_batch(100)
            # 学習の実行
            #sess.run(train_step, feed_dict={x: batch_xs, y_: batch_ys})
            gosa, a, b = sess.run([entropy_summary, cross_entropy, train_step], feed_dict={x: batch_xs, y_: batch_ys})
            if (i + 1) % 50 == 0:
                correct_prediction = tf.equal(tf.argmax(y, 1), tf.argmax(y_, 1))
                accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
                writer.add_summary(gosa, global_step = i)
                print("round", i + 1, "accuracy : ", sess.run(accuracy, feed_dict = {x: mnist.test.images, y_: mnist.test.labels}), "\n")

    writer.add_summary(gosa, global_step = i)

    """
    # モデルの評価
    # 正解データ数
    correct_prediction = tf.equal(tf.argmax(y, 1), tf.argmax(y_, 1))
    # 精度の計算
    accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
    print("round", i + 1, "accuracy : ", sess.run(accuracy, feed_dict = {x: mnist.test.images, y_: mnist.test.labels}))
    """
